Sources/AccountSDKIOSWeb/Lib/Storage/SessionStorage.swift

```diff
@@ -7,13 +7,13 @@ internal struct UserSession: Codable, Equatable {
     let updatedAt: Date
 }
 
-internal struct UserTokens: Codable, Equatable, CustomStringConvertible {
+public struct UserTokens: Codable, Equatable, CustomStringConvertible {
```

Comment:
A bit late to the party since it's already merged. 
But the memberwise-initializer in Structs is not public, so currently it's only possible to init with a decoder (which of course is also an option). But for testing purpose it would be convenient to have the initializer with the 4 parameters. That would also require the IdTokenClaims to be public.

Reply:
Sorry for missing that. I created a new PR with the necessary changes. Could you take a look? https://github.com/schibsted/account-sdk-ios-web/pull/86